libdrgn/language_c.c

```diff
@@ -19,6 +19,8 @@
 #include "type.h"
 #include "type_index.h"
 
+#define DBL_DECIMAL_DIG 17
```

Comment:
Also due to a compilation error. gcc wasn't able to find the definition of this macro even though it should technically exist in `float.h`